application/modules/statements/views/view.php

```diff
@@ -0,0 +1,173 @@
+<?php
+/*
+ * @var Client client
+ */
+?>
+
+<div id="headerbar">
+	<h1 class="headerbar-title">
+        <?php echo $statement_number; ?>
+    </h1>
+
+</div>
+
+<div id="content">
+
+    <?php echo $this->layout->load_view('layout/alerts'); ?>
+
+    <form id="statement_form" method="post" action="<?php echo site_url('statements/generate_pdf') ; // $statement_id); ?>" target="_blank">
+		<input type="hidden" name="cid" value="<?php echo $client->client_id; ?>">
+		<input type="hidden" name="sdate" value="<?php echo $statement_start_date; ?>">
+		<input type="hidden" name="edate" value="<?php echo $statement_end_date; ?>">
+
+    	<?php _csrf_field(); ?>
+    <div id="statement_form">
+			<div class="statement">
+
+				<div class="row">
+					<div class="col-xs-12 col-sm-6 col-md-5">
+
+						<h3>
+							<a href="<?php echo site_url('clients/view/' . $client->client_id); ?>">
+                            	<?php _htmlsc(format_client($client)) ?>
+                        	</a>
+						</h3>
+						<br>
+						<div class="client-address">
+                        	<?php $this->layout->load_view('clients/partial_client_address', ['client' => $client]); ?>
+                    	</div>
+                        <?php if ($client->client_phone || $client->client_email) : ?>
+                            <hr>
+                        <?php endif; ?>
+                        <?php if ($client->client_phone): ?>
+                            <div>
+                                <?php _trans('phone'); ?>:&nbsp;
+                                <?php _htmlsc($client->client_phone); ?>
+                            </div>
+                        <?php endif; ?>
+                        <?php if ($client->client_email): ?>
+                            <div>
+                                <?php _trans('email'); ?>:&nbsp;
+                                <?php _auto_link($client->client_email); ?>
+                            </div>
+                        <?php endif; ?>
+
+                </div>
+
+					<div class="col-xs-12 visible-xs">
```

Comment:
Bad indentation

Reply:
Thank you @Seb33300 . Changes done.
My editor was using tabs instead of spaces. I changed this globally for my IDE.